src/libjson/Makefile

```diff
@@ -0,0 +1,33 @@
+CFLAGS += -g
+CFLAGS += -O3
+CFLAGS += -Wall
+CFLAGS += -Werror
+CFLAGS += -Wextra
+CFLAGS += -Wno-type-limits
+
+INCLUDES += -Iinclude
+
+SOURCES += $(wildcard *.c)
+
+OBJECTS = $(SOURCES:.c=.o)
+
+HEADERS += include/json.h include/jsonprint.h
+
+all: $(OBJECTS)
```

Comment:
This works fine, but it may be a little less effort to just add the .c/.h files to the existing build scripts to avoid issues with CFLAGS not being passed correctly or the wrong compiler being used (see e.g., reference to `gcc` below).

Reply:
Oh, good point Christoph! I added that change to the new **libjson2** branch and will create a new PR from that one (I failed to successfully merge with oe_port).